src/components/entry/Login.vue

```diff
@@ -35,12 +37,12 @@
                 <section :class="{'entry-form-active': activeOption==='register'}"
                          class="entry-form entry-form-register">
                     <form id="register-form" @submit.prevent="registerUser">
-                        <input v-model="register.username" type="text" class="input-field-text"
-                               placeholder="Choose Username">
-                        <input v-model="register.password" type="password" class="input-field-text"
-                               placeholder="Enter Password">
-                        <input v-model="register.confirmPassword" type="password" class="input-field-text"
-                               placeholder="Confirm Password">
+                        <input id="newUserName" v-model="register.username" type="text"
+                               class="input-field-text" placeholder="Choose Username">
+                        <input id="newPassword" v-model="register.password" type="password"
+                               class="input-field-text" placeholder="Enter Password">
+                        <input id="confirmPassword" v-model="register.confirmPassword" type="password"
+                               class="input-field-text" placeholder="Confirm Password">
```

Comment:
```suggestion
                        <input id="new-username" v-model="register.username" type="text"
                               class="input-field-text" placeholder="Choose Username">
                        <input id="new-password" v-model="register.password" type="password"
                               class="input-field-text" placeholder="Enter Password">
                        <input id="confirm-password" v-model="register.confirmPassword" type="password"
                               class="input-field-text" placeholder="Confirm Password">
```
To keep it consistent with the other IDs, use lowercase with dashes.